Makefile

```diff
@@ -158,7 +158,7 @@ print-vm:
 
 # convenience target to setup all the bits needed for the integration tests
 # without actually running them
-prepare-check: $(NODE_MODULES_TEST) $(VM_IMAGE) test/common
+prepare-check: $(VM_IMAGE) test/common
```

Comment:
But without node_modules/ you can't run tests - they need at least `chrome-remote-interface` and `sizzle`. It *happens* to work with starter-kit, as preparing the VM always builds the webpack. But this is not true for cockpit-machines, which uses a `dist/` cache and thus running webpack for rpm/VM preparation usually does *not* happen. And we want to extend the caching to other projects at some point. But aside from that happenstance, it is still conceptually true that node_modules/ must exist for running the test.

So at least you need to move the dependency to the `check:` target, but I think given what you want to do with `prepare-check` it is required there as well.

Reply:
Hm I see, happy to drop this then.